sql/labels.sql

```diff
@@ -0,0 +1,162 @@
+  OWNER TO municipal_finance;
+
+
+
+CREATE TABLE public.cflow_labels
+(
+  code text NOT NULL,
+  label text,
+  position_in_return_form integer,
+  return_form_structure text,
+  composition text,
+  CONSTRAINT cflow_labels_primary_key PRIMARY KEY (code)
+)
+WITH (
+  OIDS=FALSE
+);
+ALTER TABLE public.cflow_labels
+  OWNER TO municipal_finance;
+
+
+CREATE TABLE public.incexp_labels
+(
+  code text NOT NULL,
+  label text,
+  position_in_return_form integer,
+  return_form_structure text,
+  composition text,
+  CONSTRAINT incexp_labels_primary_key PRIMARY KEY (code)
+)
+WITH (
+  OIDS=FALSE
+);
+ALTER TABLE public.incexp_labels
+  OWNER TO municipal_finance;
+
+
+
+CREATE TABLE public.repmaint_labels
+(
+  code text NOT NULL,
+  label text,
+  position_in_return_form integer,
+  return_form_structure text,
+  composition text,
+  CONSTRAINT repmaint_labels_primary_key PRIMARY KEY (code)
+)
+WITH (
+  OIDS=FALSE
+);
+ALTER TABLE public.repmaint_labels
+  OWNER TO municipal_finance;
+
+
+
+alter table aged_creditor_labeled add column position_in_return_form integer;
+alter table aged_creditor_labeled add column return_form_structure text;
+alter table aged_creditor_labeled add column composition text;
+
+alter table aged_debtor_labeled add column position_in_return_form integer;
+alter table aged_debtor_labeled add column return_form_structure text;
+alter table aged_debtor_labeled add column composition text;
+
+alter table capital_labeled add column position_in_return_form integer;
+alter table capital_labeled add column return_form_structure text;
+alter table capital_labeled add column composition text;
+
+alter table bsheet_labeled add column position_in_return_form integer;
+alter table bsheet_labeled add column return_form_structure text;
+alter table bsheet_labeled add column composition text;
+
+alter table cflow_labeled add column position_in_return_form integer;
+alter table cflow_labeled add column return_form_structure text;
+alter table cflow_labeled add column composition text;
+
+alter table incexp_labeled add column position_in_return_form integer;
+alter table incexp_labeled add column return_form_structure text;
+alter table incexp_labeled add column composition text;
+
+alter table repmaint_labeled add column position_in_return_form integer;
+alter table repmaint_labeled add column return_form_structure text;
+alter table repmaint_labeled add column composition text;
+
+update bsheet_labeled set position_in_return_form = subquery.position_in_return_form, return_form_structure = subquery.return_form_structure, composition = subquery.composition from (select * from bsheet_labels ) as subquery where subquery.code = item_code
+
+update aged_creditor_labeled set position_in_return_form = subquery.position_in_return_form, return_form_structure = subquery.return_form_structure, composition = subquery.composition from (select * from aged_creditors_labels ) as subquery where subquery.code = item_code;
```

Comment:
This is stuff I manually run to update the tables. Right now I deploy by doing a dump locally and applying that to prod so this is more a reminder to recreate things. We should probably clean up the sql directory